.github/actions/gradle-task/action.yml

```diff
@@ -15,9 +15,8 @@ runs:
       run: |
         if [[ ${{ inputs.is_release }} == "false"  ]];then
           echo "Workflow triggered by push to a branch"
-          export CI_VERSION=true
         fi
-        version=$(./gradlew radixCiVersion | grep radixdlt-version | cut -d: -f2)
+        version=$(./gradlew getRadixVersion | grep radixdlt-version | cut -d: -f2)
```

Comment:
Ditto regarding I imagine that we need to pass environment variables to this script? Similar to setup_version_properties?

Reply:
As above, replaced `getRadixVersion` with the version from the `setup-version-properties` action